tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1477,3 +1477,44 @@
     # cleanup
     - "Enable forceRemoveDriver on CR [1]"
     - "Delete custom resource [1]"
+
+- scenario: "Install Powermax Driver(Standalone), Enable Resiliency"
+  paths:
+    - "testfiles/storage_csm_powermax.yaml"
+  modules:
```

Comment:
Can you change `modules` to `tags` and tag these scenarios with both `"resiliency"` and `"powermax"`. Also, please verify that you have run the `powermax` e2e test suite after the changes -- the scenarios as they currently are will break e2e tests.

Reply:
Done!